ofrak_core/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to `ofrak` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+
```

Comment:
This patch edits the changeset to highlight the "so what" for the user:

```
diff --git a/ofrak_core/CHANGELOG.md b/ofrak_core/CHANGELOG.md
index ff9a0b4..5cd5e7d 100644
--- a/ofrak_core/CHANGELOG.md
+++ b/ofrak_core/CHANGELOG.md
@@ -5,33 +5,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)

 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)

-## [2.2.1](https://github.com/redballoonsecurity/ofrak/compare/ofrak-v2.2.0...ofrak-v2.2.1))
-### Fixed
-- Fix bug where initially loaded GUI resource has collapsed children [#209](https://github.com/redballoonsecurity/ofrak/pull/209)
-- Support more OpenWRT TRX files by making fewer assumptions about the partitions [#216](https://github.com/redballoonsecurity/ofrak/pull/216)
-- Fix some OS-specific problems (libmagic install, log file path) preventing OFRAK install on Windows [#239](https://github.com/redballoonsecurity/ofrak/pull/239)
-- Fix bug in GUI where "jump to offset" feature in hex view rounded up instead of down [#243](https://github.com/redballoonsecurity/ofrak/pull/243)
-
+## [2.2.1](https://github.com/redballoonsecurity/ofrak/compare/ofrak-v2.2.0...ofrak-v2.2.1) - 2023-03-08
 ### Added
-- Add keyboard shortcuts to the GUI
-- Updates to Flash components: [#195](https://github.com/redballoonsecurity/ofrak/pull/195)
-  - Flash components now support more than one occurrence of the same field type in `FlashAttributes`.
-  - `FlashOobResourceUnpacker` continues to unpack even if blocks do not perfectly align at end of the `FlashOobResource` (this is useful for real-world flash dumps).
-- Add a button to the GUI to add a new tag to a resource [#215](https://github.com/redballoonsecurity/ofrak/pull/215)
-- Add a way to sort and filter by data length or offset [#220](https://github.com/redballoonsecurity/ofrak/pull/220)
-- Add caption to ElfProgramHeader
-- Add current resource caption to GUI browser tab title
+- Add GUI features
+  - Keyboard shortcuts ([#210](https://github.com/redballoonsecurity/ofrak/pull/210))
+  - Button to add a tag to a resource ([#215](https://github.com/redballoonsecurity/ofrak/pull/215))
+  - Browser tab title contains current resource caption ([#230](https://github.com/redballoonsecurity/ofrak/pull/230))
+- Add a way to sort and filter by data length or offset ([#220](https://github.com/redballoonsecurity/ofrak/pull/220))
+- Add caption to ElfProgramHeader ([#223](https://github.com/redballoonsecurity/ofrak/pull/223))
+- Add baseline support for running pip-installing ofrak on Windows ([#228](https://github.com/redballoonsecurity/ofrak/pull/228), [#239](https://github.com/redballoonsecurity/ofrak/pull/239), [#242](https://github.com/redballoonsecurity/ofrak/pull/242), [#246]( #246))

 ### Changed
-- Tweak how errors are raised when auto-running components, so the actual root cause is not buried [#219](https://github.com/redballoonsecurity/ofrak/pull/219)
-- Show mapped resource captions on hover in the hex view [#221](https://github.com/redballoonsecurity/ofrak/pull/221)
-- Change how resources are stored to making deleting (and thus packing) much faster [#201](https://github.com/redballoonsecurity/ofrak/pull/201)
-- Use non-blocking `asyncio.create_subprocess_exec` calls in components [#53](https://github.com/redballoonsecurity/ofrak/issues/53)
+- Updates to Flash components: ([#195](https://github.com/redballoonsecurity/ofrak/pull/195))
+  - Flash components now support more than one occurrence of the same field type in `FlashAttributes`.
+  - `FlashOobResourceUnpacker` continues to unpack even if blocks do not perfectly align at end of the `FlashOobResource` (this is useful for real-world flash dumps).
+- Tweak how errors are raised when auto-running components, so the actual root cause is not buried ([#219](https://github.com/redballoonsecurity/ofrak/pull/219))
+- Show mapped resource captions on hover in the hex view ([#221](https://github.com/redballoonsecurity/ofrak/pull/221))
+- Change how resources are stored to making deleting (and thus packing) much faster ([#201](https://github.com/redballoonsecurity/ofrak/pull/201))
+- Use non-blocking `asyncio.create_subprocess_exec` calls in components ([#53](https://github.com/redballoonsecurity/ofrak/issues/53))

-### Added
-- Add stub `xattr` module to use on platforms without `xattr` support (e.g., Windows) [#228](https://github.com/redballoonsecurity/ofrak/pull/228)
+### Fixed
+- Fix bug where initially loaded GUI resource has collapsed children ([#209](https://github.com/redballoonsecurity/ofrak/pull/209))
+- Fix bug in GUI where "jump to offset" feature in hex view rounded up instead of down ([#243](https://github.com/redballoonsecurity/ofrak/pull/243))
+- Support more OpenWRT TRX files by making fewer assumptions about the partitions ([#216](https://github.com/redballoonsecurity/ofrak/pull/216))
+- Fix some OS-specific problems (libmagic install, log file path) preventing OFRAK install on Windows ([#239](https://github.com/redballoonsecurity/ofrak/pull/239))

-## [2.2.0](https://github.com/redballoonsecurity/ofrak/compare/ofrak-v2.1.1...ofrak-v2.2.0))
+## [2.2.0](https://github.com/redballoonsecurity/ofrak/compare/ofrak-v2.1.1...ofrak-v2.2.0) - 2023-02-01
 ### Fixed
 - Fix bug in data service that can cause mangled internal state [#197](https://github.com/redballoonsecurity/ofrak/pull/197)
 - Fix long-broken `OFRAK.set_id_service` [#198](https://github.com/redballoonsecurity/ofrak/pull/198)
```

Reply:
Also, probably want to add to fixed: 

- Bump reedsolo version to 1.7.0 to avoid potential [Cython-related install issues](https://github.com/tomerfiliba/reedsolomon/releases/tag/v1.7.0)

Reply:
Applying this as a git patch doesn't work for me. If you have these changes locally, can you use the github suggestions feature so I can simply apply it to the PR?